azure_functions_worker/dispatcher.py

```diff
@@ -255,8 +257,9 @@ async def _dispatch_grpc_request(self, request):
         self._grpc_resp_queue.put_nowait(resp)
 
     async def _handle__worker_init_request(self, req):
-        logger.info('Received WorkerInitRequest, request ID %s',
-                    self.request_id)
+        logger.info('Received WorkerInitRequest, '
+                    'python version %s, worker version %s, request ID %s',
+                    sys.version, __version__, self.request_id)
```

Comment:
Good addition.

Reply:
Yes, we have a log statement in https://github.com/Azure/azure-functions-python-worker/pull/815/files#diff-073d7f68306c61100a03052187af456561bb52e007a2270f03aae14f4e0f1fc6R249, `Python Worker Extension is enabled in azure.functions (SDK version)` that we can query from the Kusto.

If adding the feature flag list here, this won't work for Linux Consumption as it requires placeholder environment reload. So I still prefer adding the log inside extension.py module is a better choice.